src/desktop/components/split_test/running_tests.coffee

```diff
@@ -24,11 +24,4 @@
 # this should export empty Object
 # module.exports = {}
 
-module.exports = {
-  client_navigation_v2:
-    key: "client_navigation_v2"
-    outcomes:
-      control: 50
-      experiment: 50
-    edge: "experiment"
-}
```

Comment:
Intentionally removing the old experiment here?

Reply:
Yea, this test isn't currently running (and would need to be renamed anyway). Plus I was thinking about the currently running tests that would become part of the cache key.